homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -124,6 +124,9 @@ def _constructor(
 class RfxtrxBinarySensor(RfxtrxEntity, BinarySensorEntity):
     """A representation of a RFXtrx binary sensor."""
 
+    _attr_force_update = True
+    """We should force updates. Repeated states have meaning."""
```

Comment:
This docstring is misplaced. Use a regular code comment if you want it here or move the text into the class docstring. Also a code comment is normally put above or on the same line as the relevant code.

Reply:
Since it's cosmetic change, included this in https://github.com/home-assistant/core/pull/74927 

https://github.com/home-assistant/core/pull/74927/files#diff-c180ac5e7e0681d6b6518c4b5716439565b22e3b1eea1efe0ff3235755642df3R131